docs/pages/guides/authentication.mdx

```diff
@@ -171,6 +172,141 @@ export default function App() {
 </Box>
 {/* End Azure */}
 
+<Box
+  name="Beyond Identity"
+  createUrl="https://www.beyondidentity.com/developers/signup"
+  image={ASSETS.beyondidentity}
+>
+
+| Website                             | Provider | PKCE      | Auto Discovery |
+| ----------------------------------- | -------- | --------- | -------------- |
+| [Sign Up][c-beyondidentity] | OpenID   | Supported | Available      |
+
+[c-beyondidentity]: https://www.beyondidentity.com/developers/signup
+[c-beyondidentity-docs]: https://developer.beyondidentity.com
+[c-beyondidentity-github]: https://github.com/gobeyondidentity/bi-sdk-react-native
+[c-beyondidentity-authenticator-config]: https://developer.beyondidentity.com/docs/v1/platform-overview/authenticator-config
+
+> Beyond Identity allows developers to implement strong passwordless authentication based on public-private key pairs called "Universal Passkeys". All keys are cryptographically linked to the user and can be centrally managed using the Beyond Identity APIs. You will need a Universal Passkey before you can authenticate. 
+
+> For information on Universal Passkeys, documentation and guides visit the [developer documentation][c-beyondidentity-docs].
+
+> For a complete example app and installation instructions visit the sdk on [github][c-beyondidentity-github].
+
+<Tabs tabs={["Auth Code w/ automatic invocation", "Auth Code w/ manual invocation"]}>
+<Tab>
+Set your Beyond Identity [Authenticator Config][c-beyondidentity-authenticator-config]'s Invocation Type to <code>"Automatic"</code>. If "Automatic" is selected, Beyond Identity will automatically redirect to your application using the Invoke URL (the App Scheme or Univeral URL pointing to your application).
+{/* prettier-ignore */}
+```tsx Beyond Identity Auth Code w/ Invocation Type "Automatic"
+import * as React from 'react';
+import { makeRedirectUri, useAuthRequest, useAutoDiscovery } from 'expo-auth-session';
+import { Button } from 'react-native';
+/* @info Install the Beyond Identity Embedded SDK. This can only be used in standalone and bare workflow native apps. This SDK cannot be used in the Expo Go app. */
+import { Embedded } from '@beyondidentity/bi-sdk-react-native';
+/* @end */
+
+export default function App() {
+  // Endpoint
+  const discovery = useAutoDiscovery(`https://auth-${REGION}.beyondidentity.com/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/applications/${APPLICATION_ID}`);
```

Comment:
```suggestion
  const discovery = useAutoDiscovery(`https://auth-${region}.beyondidentity.com/v1/tenants/${tenant_id}/realms/${realm_id}/applications/${application_id}`);
```

In our docs, we usually follow the style of note capitalizing variable names in template literals.